### DISCUSSION TITLE
Docs site

### DISCUSSION THREAD

**Discussion by excid3** [excid3/madmin#185] [Ideas]

**Comment by vfonic** [comment]
Is there any, at least video tutorial of "how to use" / "the functionalities of" madmin?

I'm coming from ActiveAdmin world and am a big fan of boring rails. I'm also looking forward to use Tailwind CSS for the first time. All being said, I have no clue what this admin gem provides, apart from Resource classes which, from the first look of an untrained eye, looks like a rails scaffold. 🤷‍♂️ 

Are there any themes? Any existing open source apps where I can see some usage examples?

I greatly appreciate your work! Thanks!
